src/screens/Search/Lists/AlgoResultsList.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { View, FlatList, Text, TouchableOpacity } from 'react-native';
+import LinearGradient from 'react-native-linear-gradient';
+import ProgressiveImage from 'kitsu/components/ProgressiveImage';
+
+const ResultsList = ({
+  hits,
+  hasMore,
+  refine,
+  onPress,
+  scrollEnabled = true,
+  numColumns = 4,
+  imageSize = { h: 125, w: 91 },
```

Comment:
can these values be constants?  there are multiple pages that render poster image results that may benefit from that.